File: src/Bank.java
import java.util.ArrayList;
import java.util.List;

public class Bank {
    private List<Employee> employees;
    private List<BankAccount> bankAccounts;

    public Bank() {
        employees = new ArrayList<>();
        bankAccounts = new ArrayList<>();
    }

    public List<Employee> getEmployees() {
        return employees;
    }

    public List<BankAccount> getBankAccounts() {
        return bankAccounts;
    }

    public void addEmployee(String name, String lastName, long accNumber, char accType) {
        employees.add(new Employee(name, lastName, accNumber, accType));
        bankAccounts.add(new BankAccount(name, lastName, accNumber, accType));
    }

    public void addEmployee(String name, String lastName, long accNumber, char accType, double amount) {
        employees.add(new Employee(name, lastName, accNumber, accType, amount));
        bankAccounts.add(new BankAccount(name, lastName, accNumber, accType, amount));
    }

    public void printEmployees() {
        String employee;
        for(int i=0; i<employees.size(); i++) {
            System.out.println("------------------------------");
            employee="";
            System.out.println("Employee " + (i+1) + ":");
            employee+="Name: " + employees.get(i).getName()+"\n";
            employee+="Last name: " + employees.get(i).getLastName()+"\n";
            employee+=employees.get(i).getEmployeeAccounts();
            System.out.println(employee);
        }
        System.out.println("------------------------------");
    }

    public void printEmployee(int i) {
        String employee = "";
            System.out.println("\nEmployee " + (i) + ":");
            employee+="Name: " + employees.get(i-1).getName()+"\n";
            employee+="Last name: " + employees.get(i-1).getLastName()+"\n";
            employee+=employees.get(i-1).getEmployeeAccounts();
            System.out.println(employee);
    }

    public void printAccounts() {
        String account;
        for (int i=0; i<bankAccounts.size(); i++) {
            System.out.println("------------------------------");
            account = "";
            account += "\tAccount " + (i + 1) + ":\n";
            account += "\tAccount Owner Name: " + bankAccounts.get(i).getOwnerName() + "\n";
            account += "\tAccount Owner Last Name: " + bankAccounts.get(i).getOwnerLastName() + "\n";
            account += "\tAccount Number: " + bankAccounts.get(i).getAccountNumber() + "\n";
            account += "\tAccount Type: " + bankAccounts.get(i).getType() + "\n";
            account += "\tAccount Amount: " + bankAccounts.get(i).getAmount() + "\n";
            System.out.println(account);
        }
        System.out.println("------------------------------");
    }

    public void printAccount(int accountID) {
        String account = "";
        System.out.print("Account " + (accountID) + ":\n");
        account += "\tAccount Owner Name: " + bankAccounts.get(accountID-1).getOwnerName() + "\n";
        account += "\tAccount Owner Last Name: " + bankAccounts.get(accountID-1).getOwnerLastName() + "\n";
        account += "\tAccount Number: " + bankAccounts.get(accountID-1).getAccountNumber() + "\n";
        account += "\tAccount Type: " + bankAccounts.get(accountID-1).getType() + "\n";
        account += "\tAccount Amount: " + bankAccounts.get(accountID-1).getAmount() + "\n";
        System.out.println(account);
    }

    public void addAccount (int employeeID, long accNumber, char accType) {
            employees.get(employeeID-1).addAccount(accNumber, accType);
            bankAccounts.add(new BankAccount(employees.get(employeeID-1).getName(), employees.get(employeeID-1).getLastName(), accNumber, accType));
    }

    public void addAccount (int employeeID, long accNumber, char accType, double amount) {
            employees.get(employeeID-1).addAccount(accNumber, accType, amount);
            bankAccounts.add(new BankAccount(employees.get(employeeID-1).getName(), employees.get(employeeID-1).getLastName(), accNumber, accType, amount));
    }

}

File: bank.py
from bank_account import BankAccount
from employee import Employee


class Bank:
    __employees = []
    __bank_accounts = []

    def __init__(self):
        self.__employees = []
        self.__bank_accounts = []

    def get_employees(self):
        return self.__employees

    def get_bank_accounts(self):
        return self.__bank_accounts

    def add_employee(self, name, last_name, account_number, account_type, amount):
        self.__bank_accounts.append(BankAccount(name, last_name, account_number, account_type, amount))
        self.__employees.append(Employee(name, last_name, account_number, account_type, amount))

    def add_account(self, employee_id, account_number, account_type, amount):
        self.__employees[employee_id - 1].add_account(account_number, account_type, amount)
        self.__bank_accounts.append(BankAccount(self.__employees[employee_id - 1].get_name, self.__employees[employee_id - 1].get_last_name(), account_number, account_type, amount))

    def print_employees(self):
        for i, employee in enumerate(self.__employees, start=1):
            print("------------------------------")
            print(f"Employee {i}:")
            print(f"Name: {employee.get_name()}")
            print(f"Last name: {employee.get_last_name()}")
            print(employee.get_employee_accounts())
        print("------------------------------")

    def print_employee(self, i):
        print("\nEmployee {}:".format(i))
        print(f"Name: {self.__employees[i - 1].get_name()}")
        print(f"Last name: {self.__employees[i - 1].get_last_name()}")
        print(self.__employees[i - 1].get_employee_accounts())

    def print_accounts(self):
        for i, account in enumerate(self.__bank_accounts, start=1):
            print("------------------------------")
            print(f"\tAccount {i}:")
            print(f"\tAccount Owner Name: {account.get_owner_name()}")
            print(f"\tAccount Owner Last Name: {account.get_owner_last_name()}")
            print(f"\tAccount Number: {account.get_account_number()}")
            print(f"\tAccount Type: {account.get_type()}")
            print(f"\tAccount Amount: {account.get_amount()}")
        print("------------------------------")

    def print_account(self, account_id):
        print("Account {}:".format(account_id))
        print(f"\tAccount Owner Name: {self.__bank_accounts[account_id - 1].get_owner_name}")
        print(f"\tAccount Owner Last Name: {self.__bank_accounts[account_id - 1].get_owner_last_name()}")
        print(f"\tAccount Number: {self.__bank_accounts[account_id - 1].get_account_number()}")
        print(f"\tAccount Type: {self.__bank_accounts[account_id - 1].get_type()}")
        print(f"\tAccount Amount: {self.__bank_accounts[account_id - 1].get_amount()}")